net/ddns-scripts/Makefile

```diff
@@ -224,14 +222,24 @@ exit 0
 endef
 
 
-define Package/ddns-scripts_service/install
+define Package/ddns-scripts-services/install
 	$(INSTALL_DIR) $(1)/usr/share/ddns/services
 	$(INSTALL_DATA) ./files/usr/share/ddns/services/* \
 		$(1)/usr/share/ddns/services
+
+	# Remove special services
+	rm $(1)/usr/share/ddns/services/cloudflare.com-v4.json
+	rm $(1)/usr/share/ddns/services/freedns.42.pl.json
+	rm $(1)/usr/share/ddns/services/godaddy.com-v1.json
+	rm $(1)/usr/share/ddns/services/digitalocean.com-v2.json
+	rm $(1)/usr/share/ddns/services/no-ip.com.json
+	rm $(1)/usr/share/ddns/services/bind-nsupdate.json
+	rm $(1)/usr/share/ddns/services/route53-v1.json
+	rm $(1)/usr/share/ddns/services/cnkuai.cn.json
 endef
 
 
-define Package/ddns-scripts_cloudflare.com-v4/install
+define Package/ddns-scripts-cloudflare/install
```

Comment:
I would keep the same name and just change the '_' to '-'. People would get confuse and AFAIK there could be various version of the same script.

Reply:
> I would keep the same name and just change the '_' to '-'. People would get confuse and AFAIK there could be various version of the same script.

I do not understand that. We can add more scripts in the future. So I think it's legitimate if we keep the name more generic. The json file is still that way. So the uci parameter will not change.

Reply:
The old name directly told what service the pkg installs (without reading the description). But now that I think about it's better to have a more generic name (if more version of the same service will comes up, the generic special pkg will install all of them). Sorry for the mess.